### PR TITLE
custom rounded corners 

### PR DESCRIPTION
**Pull request by Abhimanyu121**

### CODE REVIEW

File: src/com/stag/horns/fragments/StatusbarSettings.java [hunk 1]
```diff
@@ -67,4 +86,55 @@ public void onPause() {
     public boolean onPreferenceChange(Preference preference, Object newValue) {
```
Comment (vjspranav) [inline src/com/stag/horns/fragments/StatusbarSettings.java]: Remove this duplicate boolean as you created a new one below

**Review by vjspranav — COMMENTED** [review]
You forgot adding the strings

**Comment by Abhimanyu121** [comment]
Oh wait didn't add strings , will push soon

**Comment by phenomenall** [comment]
@Abhimanyu121 OK no problem😂

File: src/com/stag/horns/fragments/StatusbarSettings.java (hunk 1 above)
Comment (Abhimanyu121) [reply src/com/stag/horns/fragments/StatusbarSettings.java]: Done :D

**Pull request closed without merging**